### PR TITLE
fixed typo and small grammatical errors in the documentation

### PR DESCRIPTION
**Pull request by tusharxoxoxo**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [ ] Does your submission pass tests?
2. [ ] Have you lint your code locally using ``cargo fmt`` command prior to submission?
3. [ ] Have you checked your code using ```cargo clippy``` command?

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Comment by generall** [comment]
Hi @tusharxoxoxo, thanks for fixes!  

`doc.md` file you changed is auto-generated, could you please fix those in original *.proto files instead?

**Comment by tusharxoxoxo** [comment]
Sure

**Comment by tusharxoxoxo** [comment]
@generall I have made those changes at *.proto files also. Can u please check and confirm that they are done rightly

File: lib/api/src/grpc/proto/collections.proto [hunk 1]
```diff
@@ -111,13 +111,13 @@ message OptimizersConfigDiff {
    */
   optional uint64 vacuum_min_vector_number = 2;
   /*
-  Target amount of segments optimizer will try to keep.
+  Target amount of segments the optimizer will try to keep.
   Real amount of segments may vary depending on multiple parameters:
 
    - Amount of stored points.
-   - Current write RPS.
```
Comment (generall) [inline lib/api/src/grpc/proto/collections.proto:118]: why?

Comment (tusharxoxoxo) [reply lib/api/src/grpc/proto/collections.proto:118]: my bad. I will fix this

**Review by generall — APPROVED** [review]

**Comment by generall** [comment]
@tusharxoxoxo could you please rebase to 'dev' branch?

**Comment by tusharxoxoxo** [comment]
okay

**Comment by tusharxoxoxo** [comment]
Have I done it correctly??

**Comment by generall** [comment]
> Have I done it correctly??
it is
`git rebase dev`

**Comment by tusharxoxoxo** [comment]
Screenshot 2023-03-13 at 14 06 50

**Comment by tusharxoxoxo** [comment]
@generall I am unable to do it. Should I fork the dev branch, make those changes, and then create a new pull request? Cause the master branch and the dev branch has too many changes and the rebase it not working properly.
Screenshot 2023-03-13 at 13 57 48

**Comment by generall** [comment]
@tusharxoxoxo you would need to create a new branch from dev and move your changes there. After that you could make a PR to qdrant:dev

**Pull request closed without merging**